mass2/core/misc.py

```diff
@@ -137,10 +152,12 @@ def launch_examples() -> None:
 
 
 def root_mean_squared(x: ArrayLike, axis: int | tuple[int] | None = None) -> float:
+    """Return the root mean square of the input along the given axis or axes."""
```

Comment:
Return the root mean square of the input along the given axis or axes. Does not subract the mean first.